bundles/binding/org.openhab.binding.insteonplm/src/main/java/org/openhab/binding/insteonplm/internal/driver/IOStream.java

```diff
@@ -103,21 +102,21 @@ private synchronized boolean reconnect() {
 
     /**
      * Creates an IOStream from an allowed config string:
-     * 
+     *
      * /dev/ttyXYZ (serial port like e.g. usb: /dev/ttyUSB0 or alias /dev/insteon)
-     * 
+     *
      * /hub2/user:password@myinsteonhub.mydomain.com:25105,poll_time=1000 (insteon hub2 (2014))
-     * 
+     *
      * /hub/myinsteonhub.mydomain.com:9761
-     * 
+     *
      * @param config
      * @return reference to IOStream
      */
 
     public static IOStream s_create(String config) {
         if (config.startsWith("/hub2/")) {
             return makeHub2014Stream(config);
-        } else if (config.startsWith("/hub/")) {
+        } else if (config.startsWith("/hub/") || config.startsWith("/tcp/")) {
```

Comment:
Will do.   I'll be able to get to it late next week

On Saturday, 9 July 2016, John Cocula <notifications@github.com> wrote:

> In
> bundles/binding/org.openhab.binding.insteonplm/src/main/java/org/openhab/binding/insteonplm/internal/driver/IOStream.java
> <https://github.com/openhab/openhab/pull/4521#discussion_r70169658>:
>
> >       * @param config
> >       * @return reference to IOStream
> >       */
> >
> >      public static IOStream s_create(String config) {
> >          if (config.startsWith("/hub2/")) {
> >              return makeHub2014Stream(config);
> > -        } else if (config.startsWith("/hub/")) {
> > +        } else if (config.startsWith("/hub/") || config.startsWith("/tcp/")) {
>
> Could you update the JavaDoc block above explaining /tcp/ like /hub/ is
> explained?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/openhab/openhab/pull/4521/files/13b9d2968cdc7799a5d587c3f153adf9af6499ac#r70169658>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AK5qXuYHUU2f-M3edGmoZO9J_YY32aJlks5qT9bAgaJpZM4JGEq1>
> .
>